articles/guides/login/_includes/_customizing-login-page.md

```diff
@@ -0,0 +1,19 @@
+## Customizing the Centralized Login Page
+
+When you integrate centralized login in your application, you users will be redirected to a default login page for your Auth0 tenant.
```

Comment:
How about:

> When you integrate centralized login in your application, you redirect the user to the `/authorize` endpoint of your Auth0 tenant. If Auth0 needs to authenticate the user, it will show the default login page.